pages/linux/rdesktop.md

```diff
@@ -0,0 +1,28 @@
+# RDESKTOP
```

Comment:
Understood, but note that in that case "SSH" is an acronym (Secure SHell), so it makes sense to capitalize it :)